rds-deployment-ha-gateway/azuredeploy.json

```diff
@@ -8,16 +8,23 @@
         "description": "Unique public DNS prefix for the deployment. The fqdn will look something like '<dnsname>.westus.cloudapp.azure.com'. Up to 62 chars, digits or dashes, lowercase, should start with a letter: must conform to '^[a-z][a-z0-9-]{1,61}[a-z0-9]$'."
       }
     },
-    "loadBalancer": {
+    "gatewayloadBalancer": {
```

Comment:
yes, there is another load balancer for ad deployed by the basic template. Just like to make it more clear. I will update it to follow the naming convention.